cpp/include/raft/comms/detail/std_comms.hpp

```diff
@@ -536,7 +537,7 @@ class std_comms : public comms_iface {
   ncclComm_t nccl_comm_;
   cudaStream_t stream_;
 
-  int *sendbuff_, *recvbuff_;
+  int* sendbuff_;
```

Comment:
I slightly prefer int32_t over int (as we use datatype_t::INT32).
And do we really need sendbuf_? Just use status_.data() instead?